components/Common/Banner/Banner.stories.tsx

```diff
@@ -0,0 +1,36 @@
+import type { Meta, StoryObj } from '@storybook/react';
+import Banner from './';
+
+const meta: Meta<typeof Banner> = {
+  title: 'Banner',
+  component: Banner,
+};
+
+export default meta;
+type Story = StoryObj<typeof Banner>;
+
+const startDate = new Date();
```

Comment:
I would extract this to a tiny function in the same file, but just to not have it coexisting with the root-level code, as it could be confusing